src/entities/base-model.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { CreateDateColumn, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm';
+
+/**
+ * 모든 엔티티의 공통 필드를 정의한 추상 클래스
+ */
+export abstract class BaseModel {
+  // insert 이전에 uuid를 생성하여 넣어준다.
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @UpdateDateColumn()

```

Comment:
[TypeORM 문서](https://typeorm.io/decorator-reference#updatedatecolumn) 에 `save` 호출 시에 시간 업데이트 해주는 특수 기능이라고 언급되어 있습니다. 수정 시 `update` 말고 `save` 를 사용해야 될 듯 합니다. (추정)
> Special column that is automatically set to the entity's update time each time you call save from entity manager or repository. You don't need to write a value into this column - it will be automatically set.

Reply:
실험 결과 `update` 로도 `@UpdatedDateColumn()` 으로 매핑해준 필드가 잘 생성되었습니다!

`update()` 전
<img width="569" alt="Screenshot 2024-08-29 at 4 35 22 PM" src="https://github.com/user-attachments/assets/740d766e-c1b3-4732-9825-a9d580626e39">

`update()` 후
<img width="528" alt="Screenshot 2024-08-29 at 4 35 57 PM" src="https://github.com/user-attachments/assets/23794dbc-940d-415b-b5b2-772ad08092b1">

SQL보시면 따로 지정해주지 않았지만 "updated_at" 에 `CURRENT_TIMESTAMP()` 로 잘 들어가고 있어 그냥 `update()` 사용하시면 될 듯 합니다~!

@kimdoyeonn @joosomi @ooheunda